From b8eab99ebecdd902574a90288b812f3e45e90f4f Mon Sep 17 00:00:00 2001
From: Viktor Hofer <viktor.hofer@microsoft.com>
Date: Wed, 18 Dec 2024 23:38:43 +0100
Subject: [PATCH] Update TargetFrameworks.props

Set NetPrevious to NetMinimum which is necessary when using the Arcade 9 SDK or an older Arcade 10 SDK without the TFM update.

Backport: https://github.com/dotnet/roslyn/pull/76506

---
 eng/targets/TargetFrameworks.props | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/eng/targets/TargetFrameworks.props b/eng/targets/TargetFrameworks.props
index 2dddaff1560..8083e84e49c 100644
--- a/eng/targets/TargetFrameworks.props
+++ b/eng/targets/TargetFrameworks.props
@@ -52,6 +52,8 @@
     -->
     <When Condition="'$(DotNetBuildSourceOnly)' == 'true' AND '$(DotNetBuildOrchestrator)' == 'true'">
       <PropertyGroup>
+        <!-- TODO until we figure out what is up with NetPrevious -->
+        <NetPrevious>$(NetMinimum)</NetPrevious>
         <NetRoslyn>$(NetCurrent)</NetRoslyn>
         <NetRoslynSourceBuild>$(NetCurrent);$(NetPrevious)</NetRoslynSourceBuild>
         <NetRoslynAll>$(NetCurrent);$(NetPrevious)</NetRoslynAll>
@@ -71,4 +73,4 @@
       </PropertyGroup>
     </Otherwise>
   </Choose>
-</Project>
\ No newline at end of file
+</Project>
